superflore/generators/bitbake/run.py

```diff
@@ -73,26 +72,37 @@ def main():
             err('Failed to file PR!')
             err('reason: {0}'.format(e))
             sys.exit(1)
-    repo_org = 'allenh1'
-    repo_name = 'meta-ros'
+    elif args.all:
+        warn('"All" mode detected... this may take a while!')
+        preserve_existing = False
+    elif args.ros_distro:
+        warn('"{0}" distro detected...'.format(args.ros_distro))
+        selected_targets = [args.ros_distro]
+        preserve_existing = False
+    if not selected_targets:
+        selected_targets = active_distros + ros2_distros
+    repo_org = 'lgsvl'
+    repo_name = 'meta-ros2'
```

Comment:
I'm more than fine with not reviewing the open embedded PR, so I'm very happy to let you guys handle it. But I would prefer that we name this `meta-ros` instead of `meta-ros2`, since it should still be making ROS 1 ebuilds.

@tfoote do you think we might consider creating a `ros/meta-ros` repo?

Reply:
That'd be great, @allenh1 @tfoote @braineniac ! I'll update it pointing to ros/meta-ros as default.

Reply:
Updated patch pointing to ```ros/meta-ros``` as the authoritative repository for OpenEmbedded.